README.md

```diff
@@ -23,27 +23,49 @@ Use the following tools to set up the project:
 - [Node.js](https://nodejs.org/) v16.0.0+
 - [npm](https://www.npmjs.com/) v7.10.0+
 
-## Usage
 
-### Install dependencies
+## Run locally
 
-To install all dependencies, run this command:
+1. Fork the repository by clicking on `Fork` option on top right of the main repository.
+
+2. Open Command Prompt on your local computer.
+
+3. Clone the forked repository by adding your own GitHub username in place of `<username>`.
+   For multiple contributions it is recommended to have [proper configuration of forked repo](https://github.com/asyncapi/community/blob/master/git-workflow.md).
 
 ```bash
```

Comment:
you need 2 tabs or 4 spaces indent here for the code snippet so once rendered, it is inlined with step 3.

Reply:
make sure all the other code/command samples follow the same